src/elastio-snap.c

```diff
@@ -1188,6 +1189,17 @@ static inline void tracer_set_memory_state(struct snap_device *dev, int error){
 	smp_mb();
 }
 
+static inline int tracer_read_cow_file_state(const struct snap_device *dev){
+	smp_mb();
+	return atomic_read(&dev->sd_cow_file_state);
+}
+
+static inline void tracer_set_cow_file_state(struct snap_device *dev, int error){
```

Comment:
This is called `error`, but it's stored in `sd_cow_file_state`.  So is it wrong to call it `cow_file_state`, or is it wrong to call this param `error`?

Reply:
It's just the same fashion as `tracer_set_fail_state` and `tracer_set_memory_state`. Again, shall I change all of them?

Reply:
The existing code is cryptic and impossible to understand therefore all new code should be cryptic and impossible to understand?  You can guess my opinion on this...